force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -39,18 +39,15 @@ public with sharing class ClassesObjectsMethods {
      * @return A list containing the length of the string, a substring of the string, and the string in lower case.
      */
     public static List<String> practiceStringMethods() {
-        // Initialize a String with 'Hello, Salesforce!'
-        
-        // Get the length of the String        
-        // Get a substring of the String of the first 5 characters
-        // Convert the String to lower case
-        
-        // Return the list with the string data
-        // In the first position add the string length
-        // In the second position add the substring
-        // In the third position add the string in lower case
-
-        return null; // Replace null with the variable you used to store the result
+        String testString = 'Hello, Salesforce!';
+        String stringLength = testString.length().toString();        
+        String substringOfTestString = testString.substring(0,5);
+        String lowerCaseTestString = testString.toLowerCase();
```

Comment:
I really like the variable names. One thing to think about as you continue programming is consider the context; because we are in this specific method, and the method is fairly short, you CAN just call it `length` or `len` even if you dare, because we can see very nearby that it is the length of the string as a string.

Reply:
Personally, I would probably name it lengthStr or something though because length is usually an integer...! so I agree with your call here.